docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,25 @@
 Version history
 ===============
 
+5.3.0-m2 (2016 November 23)
```

Comment:
One question here: are we listing each milestone separately or integrating them together? I would favor the second option as developers will typically upgrade to 5.3.0 and want to list all changes.